packages/picasso/src/PasswordInput/story/index.jsx

```diff
@@ -0,0 +1,17 @@
+import { PasswordInput } from '../PasswordInput'
+import PicassoBook from '~/.storybook/components/PicassoBook'
+
+const page = PicassoBook.section('Forms').createPage(
+  'PasswordInput',
+  'Input component for passwords'
+)
+
+page
+  .createTabChapter('Props')
+  .addComponentDocs({ component: PasswordInput, name: 'PasswordInput' })
+
+page
+  .createChapter()
+  .addExample('PasswordInput/story/Default.example.tsx', 'Default')
+  .addExample('PasswordInput/story/Disabled.example.tsx', 'Disabled')
+  .addExample('PasswordInput/story/Errored.example.tsx', 'Errored')
```

Comment:
We should skip on the legacy visual-testing, we are migrating them to Happo. Just add `// picasso-skip-visuals` to them and remove the `__image_snapshot__` png files. Without the comment the legacy framework will try to run them as a normal test
```suggestion
  .addExample('PasswordInput/story/Default.example.tsx', 'Default') // picasso-skip-visuals
  .addExample('PasswordInput/story/Disabled.example.tsx', 'Disabled') // picasso-skip-visuals
  .addExample('PasswordInput/story/Errored.example.tsx', 'Errored') // picasso-skip-visuals
```

Reply:
Good catch man! Thank you